### PR TITLE
Azure, examine and flexible LB updates

### PR DESCRIPTION
**Pull request by Shazwazza**
Updated some docs which also has to do with: http://issues.umbraco.org/issue/U4-8371

### CODE REVIEW

File: Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md [hunk 1]
```diff
@@ -38,9 +38,31 @@ The process is as follows:
...
+
+By default, Umbraco will globally set the 'base url' of the website to the address made by the first request when the AppDomain starts and it is assumed
+that your website can resolve that DNS address to itself or the main load balancer. 
+In many scenarios this is fine, but in case this is not adequate there's a few of options you can use:
+
+* set the `umbracoApplicationUrl` property in the [Web.Routing section of /Config/umbracoSettings.config](https://our.umbraco.org/Documentation/Reference/Config/umbracoSettings/)
+* or in an [`ApplicationStarting` event of an application startup handler](https://our.umbraco.org/Documentation/Reference/Events/Application-Startup) you can specify a custom delegate to return the base url for a node by setting [`ApplicationUrlHelper.ApplicationUrlProvider`](https://github.com/umbraco/Umbraco-CMS/blob/75c2b07ad3a093b5b65b6ebd45697687c062f62a/src/Umbraco.Core/Sync/ApplicationUrlHelper.cs#L21)
+* [can set your front-end(s) (non-admin server) to be explicit slave servers](https://our.umbraco.org/documentation/Getting-Started/Setup/Server-Setup/load-balancing/flexible-advanced) which means they will never be used as the master scheduler
```
Comment (Jeavon) [inline Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md]: I think this link should be relative?

Comment (Shazwazza) [inline Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md]: yup, didn't have time today though, if you're bored tonight feel free to update :) this is just a branch in the docs core

Comment (Jeavon) [inline Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md]: done, the online editor is by far the best tool for getting the relative links right :)

File: Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md [hunk 2]
```diff
@@ -59,8 +81,14 @@ You cannot share indexes between servers and since Azure Web Apps use a shared f
...
-* In ExamineSettings.config, you can add this attribute to all of your indexers and searchers: `useTempStorage="Sync"`
-* The 'Sync' setting will store your indexes in ASP.Net's temporary file folder which is on the local file system. Lucene has issues when working from a remote file share so the files need to be read/accessed locally. Anytime the index is updated, this setting will ensure that both the locally created indexes and the normal indexes are written to. This will ensure that when the app is restarted or the local temp files are cleared out that the index files can be restored from the centrally stored index files. If you see issues with this syncing process (in your logs), you can change this value to be 'LocalOnly' which will only persist the index files to the local file system in ASP.Net temp files.
+* In ExamineSettings.config, you can add these properties to all of your indexers and searchers: 
+```
+useTempStorage="Sync"
+tempStorageDirectory="UmbracoExamine.LocalStorage.AzureLocalStorageDirectory, UmbracoExamine"
```
Comment (Jeavon) [inline Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md]: Is it worth noting this is v7.3.5+ (i think)?

Comment (Shazwazza) [inline Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md]: We could sure but i guess the idea for docs is to try to minimize any reference to minimum version numbers. There is a plan to make docs much smarter when it comes to versioning but that hasn't been prioritized.

Comment (Jeavon) [inline Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md]: Thinking about it again it's probably not worth it in this case as Flexible Load Balancing is only available in 7.3+

**Pull request merged**